index.html

```diff
@@ -12,7 +12,7 @@
 <body>
   <header class="menu-wrap">
     <nav class="nav-bar">
-      <a href="#" class="brand">My Logo</a>
+      <a href="#" class="brand">Jesse</a>
       <div class="hamburger">
```

Comment:
Kindly add section tag to desktop navigation links make them links with `href=#section id`
 `<ul class="menu">
          <li>Portfolio</li>
          <li>About</li>
          <li>Contact</li>
        </ul>`